rust/src/lib.rs

```diff
@@ -112,3 +112,6 @@ pub mod plugin;
 pub mod lzma;
 pub mod util;
 pub mod ffi;
+
+//Re-export htp symbols
+pub use htp::c_api::*;
```

Comment:
Can you re explain why we need this ?

Reply:
See previous discussion: [https://github.com/OISF/suricata/pull/8335#discussion_r1063245087](here).

Short story, the htp symbols need to be exported in the rust .so so suricata can find them.

Reply:
Thanks